pyaedt/edb_core/padstack.py

```diff
@@ -469,6 +469,32 @@ def create_padstack(
         self.update_padstacks()
         return padstackname
 
+    @aedt_exception_handler
+    def duplicate_padstack(self, target_padstack_name, new_padstack_name=""):
+        """Duplicate a padstack.
+
+        Parameters
+        ----------
+        target_padstack_name : str
+            Name of the padstack to be duplicated.
+        new_padstack_name : str
+            Name of the new padstack.
+        Returns
+        -------
+
```

Comment:
right now you return a boolean. Since you give the possibility to generate automatically the padstack name, maybe you should consider returning the new padstack name once created so user knows what is the name of the new padstack

Reply:
Done